### DISCUSSION TITLE
Libary

### DISCUSSION THREAD

**Discussion by roon2010** [evilcorpltd/aTox#994] [Q&A]
Is it possible that we use aTox on our own server ?
Can you share Library setup ?

**Comment by robinlinden** [comment]
aTox (and Tox in general) doesn't really use servers per se, and stopping it from joining the wider peer-to-peer network can be tricky. :P 

If you want to build the tox4j (Toxcore) libraries, you can do as we do in CI:
https://github.com/evilcorpltd/aTox/blob/dd71f918498236acea4bb56e8cc7eebb138c1ca8/.github/workflows/ci.yaml#L77-L82

Or I just wrote a [Dockerfile][docker] that also compiles tox4j.

You can also control what nodes aTox uses to join the network by editing/replacing the [bootstrap node][nodes] json file, but if it ever speaks with any other Tox client that has joined the wider network, they'll probably start talking to each other.

What's your goal with this? :P

[nodes]: https://github.com/evilcorpltd/aTox/blob/master/atox/src/main/res/raw/nodes.json
[docker]: https://github.com/robinlinden/aTox/commit/546b8459aafed5f0ff1c2a71bc9c927d074b0af4

**Comment by roon2010** [comment]
Hello,

Thx for the email.
We have 700- 800 people working at a company that need to communicate. We looking for a chat app that can work on our server so that all data is secure.

Regards,

Ronald 

Verstuurd vanaf mijn iPhone
